redaxo/src/addons/backup/lang/de_de.lang

```diff
@@ -5,7 +5,7 @@ backup_file_error_while_delete = Datei konnte nicht gelöscht werden
 backup_no_import_file_chosen_or_wrong_version = Keine Importdatei ausgewählt oder Version falsch
 backup_no_valid_import_file = Das ist keine korrekte Importdatei
 backup_no_valid_charset = Kein passender Zeichensatz.
-backup_database_imported = Datenbankimport erfolgreich importiert
+backup_database_imported = Datenbank-Import erfolgreich importiert
```

Comment:
```suggestion
backup_database_imported = Datenbank erfolgreich importiert
```

Reply:
(Streng genommen wird nicht die Datenbank importiert, sondern werden Datenbanktabellen importiert)

Reply:
ich find's gut so

Reply:
@marcohanke Bitte oben den Vorschlag von Dirk übernehmen ("Commit suggestion"), ich hab da scheinbar kein Recht zu, "Suggested Changes" in deinen Fork zu übernehmen.